kevm_pyk/src/kevm_pyk/kevm.py

```diff
@@ -112,6 +112,10 @@ def jump_applied(pc: KInner) -> KApply:
     def pow256() -> KApply:
         return KApply('pow256_EVM-TYPES_Int', [])
 
+    @staticmethod
+    def range_uint8(i: KInner) -> KApply:
+        return KApply('#rangeUInt(_,_)_EVM-TYPES_Bool_Int_Int', [intToken(8), i])
```

Comment:
Consider making these static methods top-level functions of module `kevm_pyk.kevm` (or maybe some other module, say, `kevm_pyk.prelude`).